src/components/FilterPopup/FilterPopup.tsx

```diff
@@ -0,0 +1,112 @@
+import { useRef } from 'react';
+import styled from 'styled-components';
+import { useClickOutside, useKeyboard, KeyCode, Popup, Button, Tabs, TabContent, TabsMenu } from '@taskany/bricks';
+import { gray10, gapS, gapXs, textColor, gapM, radiusM } from '@taskany/colors';
+import { IconBulbOnOutline } from '@taskany/icons';
+
+import { Tip } from '../Tip';
+
+import { tr } from './FilterPopup.i18n';
+
+interface FilterPopupProps {
+    visible: boolean;
+    switchVisible: (val: boolean) => void;
+    filterRef: React.RefObject<HTMLSpanElement>;
+    onApplyClick: () => void;
+    activeTab?: string;
+}
+
+const StyledPopupWrapper = styled.div`
+    padding: ${gapXs};
+`;
+
+const StyledTabs = styled(Tabs)`
+    width: 500px;
+    min-height: 200px;
+
+    gap: ${gapM};
+
+    ${TabsMenu} {
+        width: 200px;
+        max-width: 100%;
+        flex-basis: auto;
+    }
+
+    ${TabContent} {
+        display: block;
+        flex: 1;
+    }
+`;
+
+const StyledPopup = styled(Popup)`
+    border-radius: ${radiusM};
+`;
+
+const StyledFilterPanelPopupFooter = styled.div`
+    display: flex;
+    padding: ${gapXs} ${gapS};
+    margin-top: ${gapXs};
+    align-items: center;
+`;
+
+const StyledActionWrapper = styled.div`
+    display: flex;
+    flex-wrap: nowrap;
+    margin-left: auto;
+    gap: ${gapS};
+`;
+
+const StyledTip = styled(Tip)`
+    color: ${gray10};
+    font-size: 0.75rem;
```

Comment:
Because, default `font-size` of `Tip` component little bit big for this component
I can move it behind `size` prop for `Tip` component in this PR or in the other issue

Reply:
> I can move it behind size prop for Tip component in this PR or in the other issue

Yes, plz!

Reply:
https://github.com/taskany-inc/issues/issues/1694